### PR TITLE
PEP8 Compliance Fixes

### PR DESCRIPTION
**Pull request by shitalmule04**
##### SUMMARY

   Bug fixed from issue #26951 

##### ISSUE TYPE

 - Bugfix Pull Request

##### COMPONENT NAME

 /config/data
##### ANSIBLE VERSION

```
2.5
```

##### ADDITIONAL INFORMATION

```

```

### CODE REVIEW

File: lib/ansible/cli/config.py [hunk 1]
```diff
@@ -1,3 +1,20 @@
+# (c) 2017, Ansible by Red Hat, Inc.
```
Comment (Akasurde) [inline lib/ansible/cli/config.py:1]: Please do not add this again as short GPL licence is added already.

File: lib/ansible/cli/config.py [hunk 2]
```diff
@@ -29,6 +46,7 @@
 class ConfigCLI(CLI):
     """ Config command line class """
 
+    VALID_ACTIONS = ("view", "dump", "list")   # TODO: edit, update, search
```
Comment (Akasurde) [inline lib/ansible/cli/config.py:49]: I think this is typo.

**Review by Akasurde — CHANGES_REQUESTED** [review]

**Comment by gundalow** [comment]
As agreed in 2017-12-07 Core meeting, we've done bulk pep8 fixes across all modules in `devel`.

This was done in https://github.com/ansible/ansible/pull/33682 and https://github.com/ansible/ansible/pull/33690 which have been merged into `devel`. because of this you will need to rebase your PR. Thanks in advance.

**Comment by gundalow** [comment]
Or perhaps this PR should be closed?

File: lib/ansible/modules/cloud/amazon/aws_kms.py [hunk 3]
```diff
@@ -126,8 +126,8 @@ def get_arn_from_kms_alias(kms, aliasname):
     key_id = None
     for a in ret['Aliases']:
         if a['AliasName'] == aliasname:
-            key_id = a['TargetKeyId']
-            break
+                key_id = a['TargetKeyId']
```
Comment (gundalow) [inline lib/ansible/modules/cloud/amazon/aws_kms.py:129]: Please revert this

File: lib/ansible/cli/config.py (hunk 1 above)
Comment (gundalow) [inline lib/ansible/cli/config.py:1]: Please revert this

**Review by gundalow — CHANGES_REQUESTED** [review]

File: lib/ansible/cli/config.py (hunk 2 above)
Comment (gundalow) [inline lib/ansible/cli/config.py:49]: Please revert this

File: lib/ansible/modules/cloud/amazon/aws_kms.py [hunk 4]
```diff
@@ -182,6 +182,10 @@ def do_grant(kms, keyarn, role_arn, granttypes, mode='grant', dry_run=True, clea
                         statement['Principal']['AWS'] = valid_entries
                         had_invalid_entries = True
 
+                    if role_arn not in statement['Principal']['AWS']:       # needs to be added.
+                        changes_needed[granttype] = 'add'
+                        statement['Principal']['AWS'].append(role_arn)
+                elif role_arn in statement['Principal']['AWS']:   # not one the places the role should be
```
Comment (sivel) [inline lib/ansible/modules/cloud/amazon/aws_kms.py:188]: This is not a PEP8 change, this is a functional change, not described as part of this pull request.

**Review by sivel — CHANGES_REQUESTED** [review]

**Comment by Akasurde** [comment]
As agreed in 2017-12-07 Core meeting, we've done bulk pep8 fixes across all modules in devel.

This was done in #33682 and #33690 which have been merged into devel. because of this you will need to rebase your PR. Thanks in advance.

**Pull request closed without merging**